_data/navigation.yml

```
@@ -145,6 +145,10 @@ docs_operate:
         url: /operate/privacy-considerations
       - title: "Performance Considerations"
         url: /operate/performance-considerations
+      - title: Backbone Operations
+        children:
+          - title: "Log Event IDs"
+            url: /operate/backbone-log-event-ids
```

I think the indentation is wrong here.
```suggestion
  - title: Backbone Operations
    children:
      - title: "Log Event IDs"
        url: /operate/backbone-log-event-ids
```